.changelog/release-v14.1.0.md

```diff
@@ -142,22 +149,30 @@ More information can be found here:
 
     Refer to [thirdparty/README.md#validating-and-testing-the-3p-api](../thirdparty/README.md#validating-and-testing-the-3p-api) on how to enabled and execute Thirdparty verification tests.
 
+8. Testing the new Bulk functionality (sdk-scheme-adapter)
+
+    For details regarding deployment and validation of simulators needed for bulk (for adoption provided in sdk-scheme-adapter) refer to [deploying Mojaloop TTK simulators](../mojaloop-ttk-simulators/README.md).
+
 ## 8. Known Issues
 
 1. [#2119 - Idempotency for duplicate quote request](https://github.com/mojaloop/project/issues/2119)
 2. [#2322 - Helm install failing with with "medium to large" release names](https://github.com/mojaloop/project/issues/2322)
-3. [#2352 - Mojaloop Helm support for Kubernetes 1.22](https://github.com/mojaloop/project/issues/2352)
-4. [#2448 - Nginx Ingress Controller v1.0.0 is incompatible with Mojaloop Helm v13.0.x releases](https://github.com/mojaloop/project/issues/2448)
-5. [#2317 - Mojaloop Helm deployments are not compatible when deployed to ARM-arch based hosts](https://github.com/mojaloop/project/issues/2317)
-6. Testing Toolkit Test Case issues causing instability/intermitant failures on Test Case Results
+3. [#2448 - Nginx Ingress Controller v1.0.0 is incompatible with Mojaloop Helm v13.0.x releases](https://github.com/mojaloop/project/issues/2448)
+4. [#2317 - Mojaloop Helm deployments are not compatible when deployed to ARM-arch based hosts](https://github.com/mojaloop/project/issues/2317)
+5. [#2740 - GP tests are failing when test currencies are used](https://github.com/mojaloop/project/issues/2740)
+6. [#3020 - Bulk prepare handler is freezing if MONGODB is disabled ](https://github.com/mojaloop/project/issues/3020)
+7. [#2892 - Disabled DFSP showing getParty info ](https://github.com/mojaloop/project/issues/2892)
```

Comment:
I am not sure this is an issue.

I believe as per our discussions, this is how it was designed. If a change is required, then it should be a feature.

Reply:
documenting it as a known issue whether by design or not.. if we resolve it one way or another, we can close it.. just showing here that it is still open.. I wanted to talk to Paul Makin and see what he thinks about this..